test/unit/test-performance.js

```diff
@@ -1309,19 +1309,6 @@ describes.realWin('PeformanceObserver metrics', {amp: true}, (env) => {
       const value = await perf.getMetric('mbv');
       expect(value).to.eq(1);
     });
-
-    describe('when API not supported', () => {
-      it('throws an error', async () => {
```

Comment:
This is still relevant. When you reject a signal, then await that signal, the promise will still reject (or throw in async/await). The only difference now is that we will not generate an out-of-band unhandled promise.

Reply:
This is still true, but on reflection I'm not sure that this behavior needs to be tested, as it is not required but rather is just a byproduct of using Signal API. @jridgewell I'd really like to get this fix in unless you feel strongly about this